[Bug, lake] mypy errors: Library stubs not installed for 'requests', more

In console:
```console
(venv) trentmc@trents-m3: ~/code/pdr-backend $ mypy ./
```

Result:
```text
pdr_backend/subgraph/core_subgraph.py:4: error: Library stubs not installed for "requests"  [import-untyped]
pdr_backend/subgraph/test/test_core_subgraph.py:2: error: Library stubs not installed for "requests"  [import-untyped]
pdr_backend/subgraph/test/test_core_subgraph.py:2: note: Hint: "python3 -m pip install types-requests"
pdr_backend/subgraph/test/test_core_subgraph.py:2: note: (or run "mypy --install-types" to install all missing stub packages)
pdr_backend/subgraph/test/test_core_subgraph.py:2: note: See https://mypy.readthedocs.io/en/stable/running_mypy.html#missing-imports
venv/lib/python3.12/site-packages/web3/_utils/request.py:24: error: Library stubs not installed for "requests"  [import-untyped]
venv/lib/python3.12/site-packages/web3/middleware/exception_retry_request.py:14: error: Library stubs not installed for "requests.exceptions"  [import-untyped]
pdr_backend/subgraph/test/test_subgraph_feed_contracts.py:2: error: Library stubs not installed for "requests"  [import-untyped]
Found 5 errors in 5 files (checked 371 source files)
```

Details:
- MacOS. Local
- This is the first time I've seen this error, it must have been introduced in the last few hours (ie after jun 18 12pm CEST)

--- a/mypy.ini
+++ b/mypy.ini
@@ -62,6 +62,9 @@ ignore_missing_imports = True
 [mypy-pytest]
 ignore_missing_imports = True
 
+[mypy-requests.*]
+ignore_missing_imports = True
+
 [mypy-sapphirepy.*]
 ignore_missing_imports = True
 

--- a/setup.py
+++ b/setup.py
@@ -47,8 +47,6 @@ install_requirements = [
     "typeguard==4.2.1",
     "types-pytz==2024.1.0.20240417",
     "types-pyYAML==6.0.12.20240311",
-    "types-requests==2.31.0.6",
-    "urllib3<2.0.2",
     "xgboost==2.0.3",
     "dash_bootstrap_components==1.6.0",
     "web3==6.19.0",

